app/models/katello/content_view_filter.rb

```diff
@@ -1,7 +1,7 @@
 module Katello
   class ContentViewFilter < Katello::Model
     include Authorization::ContentViewFilter
-    audited :associations => [:repositories]
+    audited :associations => [:repositories], :associated_with => :content_view, :except => [:name, :description]
```

Comment:
Not auditing name and description since updates to these don't require a publish.